### PR TITLE
Update Spanish translation

### PR DESCRIPTION
**Pull request by bglopez**
Updated the Spanish translation for the new strings

### CODE REVIEW

**Review by RyanNoelk — CHANGES_REQUESTED** [review]
small nit, otherwise LGTM. Gracias!

File: frontend/locale/es.json [hunk 1]
```diff
@@ -8,28 +8,28 @@
...
-  "nav.grocery_list": "Grocery List",
+  "nav.create_recipe": "Crear receta",
+  "nav.import": "Importar receta",
+  "nav.grocery_list": "Lista de compras",
   "nav.login.title": "Iniciar sesión",
   "nav.accountmenu.title": "Mi cuenta",
   "nav.accountmenu.logout": "Cerrar",
   "nav.accountmenu.admin": "Mi cuenta",
-  "nav.brand": "OpenEats",
+  "nav.brand": "Open Eats",
```
Comment (RyanNoelk) [inline frontend/locale/es.json:32]: This should be `OpenEats`

File: frontend/locale/es.json [hunk 2]
```diff
@@ -8,28 +8,28 @@
   "404.header": "Nuestro chef no ha podido preparar esta receta, te sugerimos que intentes buscar otra cosa.",
   "404.message": "No hemos podido encontrar lo que buscas.",
   "footer.credit": "Creado con {link}",
-  "footer.icon_credit": "Iconos por {link} ({ccLink})",
+  "footer.icon_credit": "Iconos diseñados por {link} ({ccLink})",
```
Comment (DJCrashdummy) [inline frontend/locale/es.json:11]: this reverts some changes from #257:
https://github.com/RyanNoelk/OpenEats/pull/257/files#diff-a60ed6296298c48b2bb1a5008fd05300
hopefully it is a correct translation...?

it should now be translated from: https://github.com/RyanNoelk/OpenEats/blob/f605cca61c049d30537b8909a91753563837d335/frontend/locale/en.json#L11

the reason and according "discussion":
https://github.com/RyanNoelk/OpenEats/pull/254/files/11781ba19e3180cfdb4db32a1520f88068f6f941#r136665863

File: frontend/locale/es.json [hunk 3]
```diff
@@ -45,32 +45,32 @@
...
   "directions.next_direction": "Próxima instrucción",
   "ingredients.add_ingredient": "Añadir ingrediente",
-  "subrecipe.add_link": "Add Link",
+  "subrecipe.add_link": "Agregar enlace",
   "directions.step_label": "Paso {step, number}",
   "directions.textarea_placeholder": "Instrucción",
   "ingredients.name_placeholder": "Nombre",
   "ingredients.quantity_placeholder": "Cantidad",
   "ingredients.measurement_placeholder": "Medida",
   "ingredients.group_placeholder": "Group (leave blank for default)",
```
Comment (DJCrashdummy) [inline frontend/locale/es.json:59]: i guess you missed this one...

**Review by DJCrashdummy — CHANGES_REQUESTED** [review]
- a translation for line 87 is still missing: https://github.com/RyanNoelk/OpenEats/pull/254#discussion_r136487440
- it would be awesome if you can also address the minor issues mentioned in my comment https://github.com/RyanNoelk/OpenEats/issues/237#issuecomment-326549952

**Comment by DJCrashdummy** [comment]
one last thing:
it would be very nice if you can take a look at line 37, if it still fits the latest minor changes at #260...
https://github.com/RyanNoelk/OpenEats/blob/5f13b447d451b068bcf7b3fef7f92aacefe805c0/frontend/locale/es.json#L37
https://github.com/RyanNoelk/OpenEats/blob/5f13b447d451b068bcf7b3fef7f92aacefe805c0/frontend/locale/en.json#L37

thank you very much @bglopez!!!

**Comment by RyanNoelk** [comment]
Merging this now as I want to get it in. I will try and fix these issues in the core PR.

**Pull request merged**